app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,221 @@
+
+One popular way to search is [search everywhere](https://www.jetbrains.com/help/idea/searching-everywhere.html) (using **Shift** twice). Search everywhere allows you to search your project files and directories, as well as your project settings and IntelliJ IDEA settings.
+
+[![Search everywhere](/assets/images/intellij-idea/search-everywhere.png)](/assets/images/intellij-idea/search-everywhere.png)
+
+Open Find in Files from the main menu using **Edit | Find | Find in Files**, or by using the shortcut **⌘⇧F** (on macOS) or **Ctrl+Shift+F** (on Windows/Linux). You can narrow down the results from **In Project** to **Module**, **Directory**, or **Scope**.
+
+[![Find in Files](/assets/images/intellij-idea/search-everywhere.png)](/assets/images/intellij-idea/find-in-files.png)
+
+## Version Control
+Once you have finished a piece of code (and the related tests and documentation), you might want to commit your code. IntelliJ IDEA offers integration with version control systems, like Git.
+
+If you haven’t set up version control for your project, you’ll need to do that first. You can enable VCS from the main menu using **VCS | Enable Version Control Integration**.
+
+[![Enable Version Control Integration](/assets/images/intellij-idea/enable-vcs-popup.png)](/assets/images/intellij-idea/enable-vcs-popup.png)
+
+In the popup that opens, select your preferred version control system.
+
+[![VCS Enable menu](/assets/images/intellij-idea/vcs-enable-menu.png)](/assets/images/intellij-idea/vcs-enable-menu.png)
+
+To use Git, you may need to [install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your machine first.
+
+To commit your code, open the commit tool window using the shortcut **⌘0** (on macOS) or **Alt+0** (on Windows/Linux), select the code to commit, write a commit message, and click **Commit** or **Commit and Push**, depending on whether you want to push to your remote repository straight away.
+
+## Evolving the project
+In the future, you might want to add functionality to your project. We’ve seen how intelliJ IDEA can help you with code suggestions and completion while writing code, running your application, adding tests and using the debugger to help figure out how code is run, refactoring code, and more.
+
+You may want to upgrade the version of Java your project uses, or any dependencies used in your project. You can manage and update your dependencies using the [Package Search](https://www.jetbrains.com/help/idea/package-search.html) plugin, see information about any known vulnerabilities in your dependencies using the [Package Checker](https://www.jetbrains.com/help/idea/package-analysis.html) plugin, and even see whether you’re using the vulnerable API using the [Vulnerable API usage](https://www.jetbrains.com/help/idea/package-analysis.html#find-vulnerable-api) inspection. You can use [the Migrate refactoring](https://www.jetbrains.com/help/idea/migrate.html) to help update your code to new versions if needed.
```

Comment:
Most beginners probably don't know what dependencies and vulnerabilities are.

Reply:
Removed the part about Package Search etc. I can add a conclusion/summary.